### DISCUSSION TITLE
index extension assignments

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#144] [General]
There are both index_genesis_assignment exists in index_genesis() and index_extension(),
<img width="659" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/c708c7aa-932d-43e0-b5f2-f8b0f7ddd07a">
<img width="592" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/1a29d1cb-26fd-48c1-95a8-68a7e6d6fda1">
So,  Can index_genesis_assignment both used for index_genesis and index_extensiion?

**Comment by dr-orlovsky** [comment]
Yes, since extension is a form of "decentralized genesis": it does not closes any single-use seals
